lib/rijksbot.rb

```diff
@@ -9,35 +9,35 @@ def initialize(twitter_api)
   end
 
   def already_replied?(tweet)
-    already_replied_to =  @twitter_api.user_timeline(count: 800).map{|t| t.in_reply_to_status_id}
+    already_replied_to = @twitter_api.user_timeline(count: 800).map(&:in_reply_to_status_id)
     already_replied_to.include?(tweet.id)
   end
 
   # create def count_twitter_max = 800
   # create def one_day_in_seconds = 86400
 
   def extract_search(tweet)
-    tweet.text.downcase.sub("@rijks_bot", "").strip
+    tweet.text.downcase.sub('@rijks_bot', '').strip
   end
 
   def random_image
-    random_words = %w(lemon pineapple banana cheese orange butterfly stilleven vanitas painting ballet unicorn heart naakt dieren)
+    random_words = %w[lemon pineapple banana cheese orange butterfly stilleven vanitas painting ballet unicorn heart naakt dieren]
     random_word = random_words.sample
-    get_image(random_word).merge({
-      "title" => "Couldn't find an image for that."
-    })
+    get_image(random_word).merge(
+      'title' => "Couldn't find an image for that."
+    )
   end
 
   def get_image(search_term)
-    api_key = ENV["RIJKSMUSEUM_API_KEY"]
-    image = Rijksmus::API::Client.new(search_term, api_key).random_image_search
+    api_key = ENV['RIJKSMUSEUM_API_KEY']
+    Rijksmus::API::Client.new(search_term, api_key).random_image_search
   end
 
   def reply_with_image(tweet, image)
-    image_url = image["webImage"]["url"]
-    image_title = image["title"]
-    screen_name =  "@" + tweet.user.screen_name
-    text = screen_name + " " + image_title
-    @twitter_api.update_with_media(text, open(image_url), in_reply_to_status_id: tweet.id)
+    image_url = image['webImage']['url']
+    image_title = image['title']
+    screen_name = '@' + tweet.user.screen_name
+    text = screen_name + ' ' + image_title
```

Comment:
I must admit I got used to using only double quotes. Will consider single quotes if there is a reason for it :)

Reply:
Reasons are again, a bit less memory,also then, when you see double quotes is easy to see that may be some interpolation.. By default, rubocop, is switching double quotes for single ones. He is happy only when you need double quotes. I am the other way around. I only use double quotes when I need it. 

```ruby
"Rubocop says, I'm  with this because there is a tilde"
"Rubocop says, I am fine with this because #{@this_is_a_string_interpolation}"
```

You can decide wich approach to choose but in case you decide for double quotes instead my approach, I recommend to add this line in the .rubocop.yml
```yml
Style/StringLiterals:
  EnforcedStyle: double_quotes
```
** the two spaces indentation in "Enforce." is very important here.

And then check rubocop new complaints just running in the shell:
```bash
rubocop
```
Once then, for fixing this refactoring changes you can use the next command
```bash
rubocop -a
```
the "-a" is from auto-correct, is a shortcut flag of "rubocop --auto-correct"
Then you will see that this little complaints are gone. I know! It's so magic!
From simple and easy refactoring Rubocop can help you without create any pain and clean code with your favorite default ruby style guide. This is why is one my favorite gems :).

Reply:
Let's stick with rubocop and you!